### ISSUE TITLE
Learning styles

### ISSUE THREAD

**Issue by ArcoMul** [HackYourFuture-CPH/teacher-curriculum-exercise-tracker#4]
# Learning styles

Link to exercise: https://hyfbe.gitbook.io/teacher-curriculum/exercises/learning-styles#exercises

---

**How many learning styles do you think there are?**

From what I just read I conclude there are an infinite amount of learning styles.

<br/>

**What learning styles are these?**

- Reading/writing
- Visual learning
- Auditory learning
- Doing by example

As the science says these are not distinct groups. Still I think that there is a truth in these different ways of up taking information and that some people prefer one over the other. Which method students prefer might differ by topic and might even differ every other day. And often a combination might be best I imagine.

<br/>

**Select a topic you have taught before. Prepare a description of how you would teach the topic for each of the learning styles**

Let's say I'm teaching Git. I guess my current teach method has some styles in it, but let's say in an ideal scenario where I want to incorporate all styles:

>I'd take off by summarizing the abstract concepts of Git, what is Git, what is its use, why is it being used (auditory), while also writing down the different things I say in a short list (reading). I'd give some examples of what Git can be compared to (doing by example?). After finishing this short introduction I'd show a quick example of how to do basic thing with Git (visually), and ask the students to do something else simple by themselves (doing by example). After having done all that I would summarize what Git is, and everything we did (auditory).

I hope that in that manner, by touching all the different learning styles students could take up this information it will stick one way or the other.

<br/>

**What learning style do you think you yourself are? How would you teach the topic above to yourself?**

I guess I mainly learn by doing. Although I would also define myself as a visual learning I know that when I look at graphs describing Git branches and merging and what not, I get only more confused.

If I remember correctly when learning Git I first read some explanation of what Git is. Which left me confused. Then I probably searched for a comparison between SVN and Git. This most likely still left me confused. Following up on that I started using it. Which made me fully understand Git in the end. While in hindsight probably all of the before were needed to fully understand its workings.

**Comment by benna100** [comment]
I agree with there not being a set of specific styles but more combination of all learning styles. 

As you also write trying to hit different learning styles is probably a good approach. Or at least not only having one learning style is a good approach. 

I'm also super much learning by example!

**Comment by Toinne** [comment]
Love the approach indeed the combination of all in a "lecture" and a mix seems best to me too.
